README.md

```diff
@@ -1,94 +1,20 @@
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-9-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
+<!-- Read The Docs Badge -->
+[![Documentation Status](https://readthedocs.org/projects/novelcovid/badge/?version=latest)](https://novelcovid.readthedocs.io/en/latest/?badge=latest)
+<!-- Read The Docs Badge End -->
+
 # Join our Server!
 [![Discord server](https://discordapp.com/api/guilds/689535536934813823/embed.png?style=banner4)](https://discord.gg/EvbMshU)
 
 # Partners with PrimedHosting
 Great and affordable prices! Starting at $0.75. Get started [here](https://primedhosting.com/NovelCovid)
 Join the [Discord server](https://primedhosting.com/discord)
 
-# API
-API for Current cases and more stuff about COVID-19 or the Novel Coronavirus Strain
-https://corona.lmao.ninja/
-
-# Endpoints v2
-|  GET Request  | Output  |
-| ------------ | ------------ |
-| https://corona.lmao.ninja/v2/historical | Get historical data from the start of 2020. (JHU CSSE GISand Data) |
-| https://corona.lmao.ninja/v2/historical/{country-name} | Get historical data from the start of 2020 for a specific country. (JHU CSSE GISand Data) |
-|  https://corona.lmao.ninja/v2/jhucsse | Return data from the Johns Hopkins CSSE Data Repository (Country, province, confirmed, death, recovered) |
-
-# Endpoints v1
-|  GET Request  | Output  |
-| ------------ | ------------ |
-|  https://corona.lmao.ninja/all | Returns all total cases, recovery, and deaths. |
-|  https://corona.lmao.ninja/countries | Returns data of all countries that has COVID-19 |
-|  https://corona.lmao.ninja/countries?sort={parameter} | Returns data of each country sorted by the parameter |
-|  https://corona.lmao.ninja/countries/{country-name} | Returns data of a specific country. If an exact name match is desired pass ?strict=true in the query string |
-|  https://corona.lmao.ninja/states | Returns all United States of America and their Corona data |
-|  https://corona.lmao.ninja/jhucsse | **DEPRECATED USE V2 ENDPOINT** Return data from the Johns Hopkins CSSE Data Repository (Provinces and such) |
-| https://corona.lmao.ninja/historical | **DEPRECATED USE V2 ENDPOINT** Get historical data from the start of 2020. (JHU CSSE GISand Data) |
-| https://corona.lmao.ninja/historical/{country-name} | **DEPRECATED USE V2 ENDPOINT** Get historical data from the start of 2020 for a specific country. (JHU CSSE GISand Data) |
```

Comment:
I think these should stay in README for quick access by people just wanting to use the endpoint, which is separate from the npm package.

Reply:
I totally understand! I'll get that committed back to normal by tonight.